lang/en/logstore_xapi.php

```diff
@@ -123,3 +123,5 @@
 $string['replayevent'] = 'Replay event';
 $string['confirmresendeventsheader'] = 'Resend events';
 $string['confirmresendevents'] = 'You are about to send {$a->count} record(s) to the queue for reprocessing.<br>Do you wish to continue?';
+$string['resendevents:success'] = 'Events are set for reprocessing successfully.';
```

Comment:
Change to 
"Events successfully sent for reprocessing"

Reply:
great, thx.